projects/dsp-ui/src/lib/core/session.service.spec.ts

```diff
@@ -0,0 +1,205 @@
+import { async, TestBed } from '@angular/core/testing';
+import {
+    ApiResponseData,
+    AuthenticationEndpointV2,
+    CredentialsResponse,
+    KnoraApiConfig,
+    MockUsers,
+    UsersEndpointAdmin
+} from '@dasch-swiss/dsp-js';
+import { of } from 'rxjs';
+import { DspApiConfigToken, DspApiConnectionToken } from './core.module';
+import { Session, SessionService } from './session.service';
+
+describe('SessionService', () => {
+    let service: SessionService;
+
+    beforeEach(async(() => {
+        const dspConfSpy = new KnoraApiConfig('http', 'localhost', 3333, undefined, undefined, true);
+
+        const dspConnSpy = {
+            admin: {
+                usersEndpoint: jasmine.createSpyObj('usersEndpoint', ['getUser'])
+            },
+            v2: {
+                auth: jasmine.createSpyObj('auth', ['checkCredentials', 'login'])
+            }
+        };
+
+        TestBed.configureTestingModule({
+            imports: [],
+            providers: [
+                {
+                    provide: DspApiConfigToken,
+                    useValue: dspConfSpy
+                },
+                {
+                    provide: DspApiConnectionToken,
+                    useValue: dspConnSpy
+                }
+            ]
+        });
+        service = TestBed.inject(SessionService);
+    }));
+
+    // mock localStorage
+    beforeEach(() => {
+        let store = {};
+
+        spyOn(localStorage, 'getItem').and.callFake(
+            (key: string): string => {
+                return store[key] || null;
+            }
+        );
+        spyOn(localStorage, 'removeItem').and.callFake(
+            (key: string): void => {
+                delete store[key];
+            }
+        );
+        spyOn(localStorage, 'setItem').and.callFake(
+            (key: string, value: string): string => {
```

Comment:
`setItem` does not return a value, see https://developer.mozilla.org/en-US/docs/Web/API/Storage/setItem

Reply:
`setItem` has a return type of `void`,  `store[key] = value` is enough.